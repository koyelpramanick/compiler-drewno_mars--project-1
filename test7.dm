while (guy == false) {
    num++;
    butt = "this is a correct string"
} //this is a right paren
"bad escape oh no! \o" 67
today I don't feel like doing any work at all!
"let's fail to terminate this string
24Kmagic 600000000000000
"bad escape \q and unterminated...
//i hope that's all