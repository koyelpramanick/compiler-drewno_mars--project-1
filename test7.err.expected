FATAL [5,1]-[5,23]: String literal with bad escape sequence detected
FATAL [7,1]-[7,38]: Unterminated string literal detected
FATAL [8,10]-[8,25]: Integer literal overflow
FATAL [9,1]-[9,36]: Unterminated string literal with bad escape sequence detected
